lib/inspec/cli.rb

```diff
@@ -330,6 +323,7 @@ def detect
   option :distinct_exit, type: :boolean, default: true,
     desc: "Exit with code 100 if any tests fail, and 101 if any are skipped but none failed (default).  If disabled, exit 0 on skips and 1 for failures."
   def shell_func
+    require "inspec/resources" # TODO: remove me if possible!
```

Comment:
I'm not sure we can - we definitely need it for tab completion and help.  If we had multithreading, we could get to a prompt early and then load the heavy stuff while the user types...

Reply:
I'm curious how many users use shell, autocomplete, or help =)

I think so long as we tell people we are going to break this functionality for a limited time while are making things faster this would be fine.

Reply:
I worked this out too.